integration/configs/browserCompatibilityTest/content_share_screen_capture_test.config.json

```diff
@@ -0,0 +1,50 @@
+{
+  "`": 1,
+  "name": "content_share_screen_capture_test.config.json %ts",
+  "grids": [],
+  "tests": [
+    {
+      "name": "Content Share Screen Capture %ts",
+      "description": "Screen sharing using content share",
+      "tupleSize": 1,
+      "noOfThreads": 1,
+      "testImpl": "ContentShareScreenCapture.js",
+      "payload": {
+        "url": "http://localhost:8080/",
+        "retry": 2
+      }
+    }
+  ],
+  "clients": [
```

Comment:
Any reason why we do not run against Firefox / Safari here?

Reply:
This is an existing issue and needs to be looked into. Currently tracking this in backlog.